### ISSUE TITLE
[FR] Be able to configure shared directory name

### ISSUE THREAD

**Issue by adamj88** [timkelty/shipit-shared#44]
The `shared/` directory name is hard coded here: https://github.com/timkelty/shipit-shared/blob/master/lib/init.js#L21

Would it be possible to be able to set this in the configuration, falling back to `shared/` to prevent any BC break?

**Comment by timkelty** [comment]
This already should be configurable...
Try settings `shipit.config.shared.basePath` - it _defaults_ to `path.join(shipit.config.deployTo, 'shared')`, but can be set manually.

**Comment by timkelty** [comment]
Unless you specifically just wanted to change the directory _name_ and not the whole path...

**Comment by adamj88** [comment]
Thats correct, my suggestion is to be able to configure the directory name independently.

So `shipit.config.deployTo` as the path, and a separate configuration for the `'shared'` string, `shipit.config.dirName` possibly?

I've got a similar issue open on shipit-deploy: https://github.com/shipitjs/shipit-deploy/issues/128 - the net result should be that you're able to configure all directory names.
